backend/src/controllers/booking.controllers.ts

```diff
@@ -1,12 +1,22 @@
+import jwt, { JwtPayload } from 'jsonwebtoken';
+
+import * as fs from 'fs';
 import { Router, RouterOptions } from 'express'
 import IBookingService from '../services/booking.service'
 import { validateRoom, validateUserName, validateBikeNumbering } from '../models/validators'
 import { BookingDTO, BookingStatusDTO } from '../dto/booking.dto'
 import { Booking, BookingStatus, BookingType } from '../models/booking.model'
 import { getLogger } from '../logger'
-import { getDecodedToken, validateAccountPermission } from '../utils/auth';
+import { generatePublicAsyncToken, getDecodedToken, validateAccountPermission } from '../utils/auth';
 import { AccountType } from '../models/account.model'
 
+let publicJwtKey: string = ""
+
+if (process.env.PUBLIC_JWT_KEY_FILE) {
+  publicJwtKey = fs.readFileSync(process.env.PUBLIC_JWT_KEY_FILE, 'utf8')
+} else if (process.env.PUBLIC_JWT_KEY) {
+  publicJwtKey = process.env.PUBLIC_JWT_KEY;
+}
```

Comment:
Why we need this public key reading here? I thought this was already loaded on `utils/auth.ts`

Reply:
Good point! I forgot that just one import is enough for everyone.
Done!